README.md

```diff
@@ -60,16 +77,30 @@ npm run build
 ```
 
 Install the socket server node packages. Open a new terminal, `cd` to `cla_frontend` and run:
+
+on Linux
 ```
 cd cla_frontend/cla_socketserver/
+```
+on Windows
+```
+cd cla_socketserver
+```
+then
+```
 npm install
 node app.js
 ```
+Leave this running and return to the previous window/tabtab, start the runserver. Don't forget to keep the backend server running on port 8000:
 
-In the main tab, start the runserver. Don't forget to keep the backend server running on port 8000:
+on Linux
 ```
 ./manage.py runserver 8001
 ```
+on Windows 
+```
+python ./manage.py runserver 8001
```

Comment:
As I said, this version works on both, so let's keep it simple and use it for both